### PR TITLE
Improve new search bar accessibility

### PR DESCRIPTION
**Pull request by matijs**
Add a (hidden) label for the new SearchBar component.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -77,6 +77,12 @@ export * from './components/tooltip';
 export * from './components/user_account_menu';
 export * from './components/video_player';
 
+export * from './components/input';
+export { InputText } from './components/input/input_text';
+export { SearchBar } from './components/search_bar';
+// once icons are in their own package this will be removed
+export * from './components/icons';
```
Comment (ianpaschal) [inline src/index.ts]: Icons are components too; I'd prefer to avoid ninety billion PRs and release and npm installs required when we make changes.

Comment (matijs) [reply src/index.ts]: Good, should anything change, then there'll only be a few so we can deal with that.

**Comment by xli12** [comment]
The content of the PR is fine with me, but there's one thing not very clear: the rationale of this improvement. What will be the use case of this `visually-hidden` style? By introducing a `UUID` field, what kind of issues can we possibly address?

**Comment by matijs** [comment]
@xli12 all input fields need a label but sometimes design does not think that making that label visible is necessary, in those cases we should visually hide it. Using a hardcoded value to link a `<label>` to an `<input>` would break if component appeared more than once on a page, therefore a UUID is used.

File: src/index.ts (hunk 1 above)
Comment (ianpaschal) [reply src/index.ts]: My point being that I disagree that icons should be a different package.

File: .storybook/webpack.config.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ module.exports = {
                 test: /\.tsx?$/,
                 loaders: [
                     'awesome-typescript-loader',
-                    'react-docgen-typescript-loader',
+                    // 'react-docgen-typescript-loader',
```
Comment (matijs) [inline .storybook/webpack.config.js]: @sedwards2009 found that disabling this speeds up the build no end. Is anybody using the functionality provided by [react-docgen-typescript-loader](https://github.com/strothj/react-docgen-typescript-loader)?

File: src/stylesheets/_accessibility.scss [hunk 3]
```diff
@@ -0,0 +1,10 @@
+@mixin visually-hidden {
+    border: 0;
```
Comment (ImreBognarUltimaker) [inline src/stylesheets/_accessibility.scss]: Isn't there a less hacky way to do this?

Comment (matijs) [reply src/stylesheets/_accessibility.scss]: This has been the de facto way to visually hide content for quite some time.

File: src/index.ts (hunk 1 above)
Comment (matijs) [reply src/index.ts]: Fine, we'll discuss that when we get there :) This PR still contains the icons in case you had missed that.

**Pull request merged**